src/app/containers/Article/__snapshots__/index.test.jsx.snap

```diff
@@ -9,7 +9,6 @@ exports[`ArticleContainer Component should render Persian article correctly 1`]
   >
     <Header />
     <Metadata
-      amp={false}
```

Comment:
The props need to be updated in the tests for the Article container - it is still `amp` and so is not being passed to the Metadata component: https://github.com/BBC-News/simorgh/blob/3ecf8f91f1d7704aebf8e6a49cd3f27411836eda/src/app/containers/Article/index.test.jsx#L10